From: Jan Engelhardt <jengelh@inai.de>
Date: 2023-11-01 14:19:29.990929358 +0100
References: https://github.com/kraflab/dsda-doom/issues/434

[    6s] src/CMakeFiles/dsda-doom.dir/build.make:3952: warning: overriding recipe for target 'dsda-doom'
[    6s] src/CMakeFiles/dsda-doom.dir/build.make:3943: warning: ignoring old recipe for target 'dsda-doom'
[    6s] src/CMakeFiles/dsda-doom.dir/build.make:3952: warning: overriding recipe for target 'dsda-doom'
[    6s] src/CMakeFiles/dsda-doom.dir/build.make:3943: warning: ignoring old recipe for target 'dsda-doom'
[    6s] make[2]: *** No rule to make target 'optimized', needed by 'dsda-doom'.  Stop.
[    6s] make[1]: *** [CMakeFiles/Makefile2:172: src/CMakeFiles/dsda-doom.dir/all] Error 2

---
 prboom2/cmake/FindLibMad.cmake |    2 --
 1 file changed, 2 deletions(-)

Index: dsda-doom-0.27.2/prboom2/cmake/FindLibMad.cmake
===================================================================
--- dsda-doom-0.27.2.orig/prboom2/cmake/FindLibMad.cmake
+++ dsda-doom-0.27.2/prboom2/cmake/FindLibMad.cmake
@@ -59,13 +59,13 @@ select_dll_configurations(LibMad)
 find_library(
   LibMad_LIBRARY_RELEASE
   NAMES mad
-  PATH_SUFFIXES ../../lib
+  PATH_SUFFIXES ${CMAKE_CURRENT_BINARY_DIR}/../../lib
 )
 
 find_library(
   LibMad_LIBRARY_DEBUG
   NAMES mad
-  PATH_SUFFIXES ../debug/lib
+  PATH_SUFFIXES ${CMAKE_CURRENT_BINARY_DIR}/../debug/lib
 )
 
 include(SelectLibraryConfigurations)
